src/plugins/home/server/services/sample_data/data_sets/util.ts

```diff
@@ -5,59 +5,74 @@
 
 import { SavedObject } from 'opensearch-dashboards/server';
 
-export const appendDataSourceId = (id: string) => {
```

Comment:
Seems this logic has gone?

Reply:
Any updates on this comment?

Reply:
The `appendDataSourceId` has been added back. To support workspace, the workspaceId has been added to the `appendDataSourceId` result function. It will add workspace id prefix at the return value of `getDataSourceIntegratedDashboard` and `getDataSourceIntegratedDefaultIndex` functions.